src/main/java/acme/entities/Audit.java

```diff
@@ -0,0 +1,53 @@
+
+package acme.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
+import org.hibernate.validator.constraints.Length;
+
+import acme.framework.data.AbstractEntity;
+import acme.roles.Auditor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+public class Audit extends AbstractEntity {
+
+	// Serialisation identifier -----------------------------------------------
+	protected static final long	serialVersionUID	= 1L;
+
+	// Attributes -------------------------------------------------------------
+	@Column(unique = true)
+	@NotBlank
+	@Pattern(regexp = "[A-Z]{1,3}\\d\\d{3}", message = "default.error.conversion")
+	protected String			code;
+
+	@NotBlank
+	@Length(max = 100)
+	protected String			conclusion;
+
+	@NotBlank
+	@Length(max = 100)
+	protected String			strongPoints;
+
+	@NotBlank
+	@Length(max = 100)
+	protected String			weakPoints;
+
+	protected String			mark;
+	// Derived attributes -----------------------------------------------------
+
+	// Relationships ----------------------------------------------------------
```

Comment:
Falta relación many to one hacia la entidad course, ya que una auditoría es un documento que registra auditorías de un curso